lms/djangoapps/discussion_api/views.py

```diff
@@ -105,16 +110,21 @@ class ThreadViewSet(_ViewMixin, DeveloperErrorViewMixin, ViewSet):
         * following (optional): A boolean indicating whether the user should
             follow the thread upon its creation; defaults to false
 
+    **PATCH Parmaeters**:
+
+        topic_id, type, title, and raw_body are accepted with the same meaning
+        as in a POST request
+
```

Comment:
thread_id cannot be provided in the body. The thread_id parameter in the partial_update method comes from the URL.